test/jdk/javax/swing/JMenuItem/MenuItemTest/MenuItemTestHelper.java

```diff
@@ -168,4 +143,23 @@ private static JMenu createSomeIconsMenu(Icon myIcon, Icon myIcon2) {
 
         return someIcons;
     }
+
+    private record ColoredIcon(Color color, int width,
+                               int height) implements Icon {
+        @Override
+            public void paintIcon(Component c, Graphics g, int x, int y) {
```

Comment:
```suggestion
        @Override
        public void paintIcon(Component c, Graphics g, int x, int y) {
```
You have reduce indentation of the entire `ColoredIcon` class.